code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -329,7 +329,7 @@
 		playsound(loc, 'sound/effects/glass_step.ogg', 50, 1)
 		if(ishuman(AM))
 			var/mob/living/carbon/human/H = AM
-			if(!(PIERCEIMMUNE in H.dna.species.specflags))
+			if(!(PIERCEIMMUNE in H.dna.species.specflags) && !buckled_mob,/obj/structure/bed/chair/withwheels)
```

Comment:
change && to ||
and put !istype(buckled_mob,/obj/structure/bed/chair/withwheels)